pkgs/development/python-modules/bitmath/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchPypi, isPy36, progressbar, mock }:
+
+buildPythonPackage rec {
+  pname = "bitmath";
+  version = "1.3.1.2";
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.